### DISCUSSION TITLE
我的代理是用xray在docker里跑的，通过这样的代理会有这样的提示：你发起连接的系统与你的电脑系统不同

### DISCUSSION THREAD

**Discussion by hzdlive** [jason5ng32/MyIP#220] [Q&A]
你发起连接的系统与你的电脑系统不同，你可能正在使用代理。尽管系统判断不一定准确，但可以肯定两个系统并不相同。 你的电脑系统是： Windows. 你发起连接的系统可能是： Linux-based OS

怎么修改xray的配置才能避免被检测出这个呢？

**Comment by jelly21fish** [comment]
我使用指纹浏览器（免费的，可能功能不全）将操作系统指纹改为Linux，结果这里仍然显示不一致：Linux和Linux-based OS不同！真的很想知道怎么解决，果然不止我一个受此困扰。想在推上询问Jason大神，可惜不接受普通用户的私信，希望能在此得到回答🙏
